certbot-dns-google/certbot_dns_google/dns_google.py

```diff
@@ -183,3 +204,22 @@ def _find_managed_zone_id(self, domain):
 
         raise errors.PluginError('Unable to determine managed zone for {0} using zone names: {1}.'
                                  .format(domain, zone_dns_name_guesses))
+
+    @staticmethod
+    def get_project_id():
+        """
+        Query the google metadata service for the current project ID This only works on
+        Google Cloud Platform and raises a ServerNotFoundError if metadata.google.internal
+        is not available
```

Comment:
Thanks for writing meaningful documentation here! [PEP 257](https://www.python.org/dev/peps/pep-0257) (which is included in both of our [style guides](https://certbot.eff.org/docs/contributing.html#coding-style)) says that multi-line docstrings should have a one line summary, a blank line, and then a longer description. Ideally we should follow this convention here as well.